compiler/z/codegen/FPTreeEvaluator.cpp

```diff
@@ -459,7 +459,8 @@ convertToFixed(TR::Node * node, TR::CodeGenerator * cg)
    else  // Signed Target Path
       {
       //1) Reset targetRegister
-      generateRIInstruction(cg, TR::InstOpCode::LA, node, targetRegister, 0);
+
+      cursor = generateRRInstruction(cg, TR::InstOpCode::XGR, node, targetRegister, targetRegister);
```

Comment:
A different pull request removed the cursor variable which is causing compilation errors, cursor doesn't have to be set and can safely be removed here.